wren/compute/prepare.go

```diff
@@ -171,7 +171,7 @@ func (m *Manager) executePrepare(ctx context.Context, owner *ent.Owner, prepareJ
 		prepareReq := &v1alpha.PrepareDataRequest{
 			SourceData:       sourceData,
 			Config:           computeTable.Config,
-			OutputPathPrefix: ConvertURIForCompute(prepareOutputURI),
+			OutputPathPrefix: prepareOutputURI,
```

Comment:
if sparrow now supports all the standard file URIs correctly (`file:///, s3://, gs://, https://`) then there are probably several other places where `ConvertURIForCompute` and its partner `ConvertURIForManager` should be removed.

Ideally both functions should be removed in their entirety.